util-taglib/src/com/liferay/taglib/servlet/JspWriterHttpServletResponse.java

```diff
@@ -40,6 +41,21 @@ public JspWriterHttpServletResponse(PageContext pageContext) {
 	public ServletOutputStream getOutputStream() {
 		return new ServletOutputStream() {
 
+			@Override
+			public boolean isReady() {
+				return true;
+			}
+
+			/**
+			* @since Servlet 3.1
+			*/
+			@Override
+			public void setWriteListener(WriteListener listener) {
```

Comment:
@holatuwol The requirements from the [Javadoc for this method](https://docs.oracle.com/javaee/7/api/javax/servlet/ServletOutputStream.html#setWriteListener-javax.servlet.WriteListener-) have not been met either.